rocky/katalogus/views/mixins.py

```diff
@@ -71,11 +71,12 @@ def dispatch(self, request, *args, **kwargs):
         return super().dispatch(request, *args, **kwargs)
 
     def is_required_field(self, field: str) -> bool:
+        """Check whether this field should be required, defaults to False."""        
         return self.plugin_schema and field in self.plugin_schema.get("required", [])
 
     def is_secret_field(self, fields: dict) -> bool:
         """Check whether this field should be secret, defaults to False."""
-        return fields.get("hide_value_in_interface", False)
+        return self.plugin_schema and field in self.plugin_schema.get("secret", [])
```

Comment:
This is now almost a copy of `is_required_field`, maybe a separate function to call would be cleaner

Reply:
It is, and yes, maybe we can refactor this, but lets keep this simple for now.